modules/volume_access.cft.yaml

```diff
@@ -394,13 +395,54 @@ Resources:
                     Condition:
                       StringEqualsIgnoreCase:
                         "aws:ResourceTag/CreatedBy": "Sysdig" 
+          AdministrationRole:
+            Type: AWS::IAM::Role
+            Properties:
+              RoleName: !Sub sysdig-secure-scanning-stackset-administration-${NameSuffix}
+              AssumeRolePolicyDocument:
+                Version: 2012-10-17
+                Statement:
+                - Effect: Allow
+                  Principal:
+                    Service: cloudformation.amazonaws.com
+                  Action:
+                  - sts:AssumeRole
+              Policies:
+              - PolicyName: !Sub sysdig-secure-scanning-stackset-execution-${NameSuffix}
+                PolicyDocument:
+                  Version: 2012-10-17
+                  Statement:
+                  - Effect: Allow
+                    Action:
+                    - sts:AssumeRole
+                    Resource:
+                    - !Sub arn:aws:iam:::role/sysdig-secure-scanning-stackset-execution-${NameSuffix}
+          ExecutionRole:
+            Type: AWS::IAM::Role   
+            Properties:
+              RoleName: !Sub sysdig-secure-scanning-stackset-execution-${NameSuffix}
+              AssumeRolePolicyDocument:
+                Version: 2012-10-17
+                Statement:
+                - Effect: Allow
+                  Principal:
+                    AWS:
+                    - !GetAtt AdministrationRole.RoleId
+                  Action:
+                  - sts:AssumeRole
+              ManagedPolicyArns:
+              - arn:aws:iam::aws:policy/AWSKeyManagementServicePowerUser
+              - arn:aws:iam::aws:policy/AWSCloudFormationFullAccess
   OrganizationKMSKeyStackSet:
     Type: AWS::CloudFormation::StackSet
     Condition: IsOrganizational
+    DependsOn: OrganizationRoleStackSet
     Properties:
       StackSetName: !Sub sysdig-secure-scanning-organization-kmskey-${NameSuffix}
       Description: IAM Role used to create KMS Keys to scan organization accounts/regions
-      PermissionModel: SERVICE_MANAGED
+      AdministrationRoleARN: !Sub arn:aws:iam:::role/sysdig-secure-scanning-stackset-administration-${NameSuffix}
+      ExecutionRoleName: !Sub sysdig-secure-scanning-stackset-execution-${NameSuffix}
+      PermissionModel: SELF_MANAGED
```

Comment:
shouldn't this remain SERVICE_MANAGED?

Reply:
I was attempting different approaches. The latest implementation does use service_managed stacksets for OUs X regions in member accounts